homeassistant/components/airzone/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "airzone",
+  "name": "Airzone",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/airzone",
+  "requirements": ["aioairzone==0.0.2"],
+  "codeowners": ["@Noltari"],
+  "iot_class": "local_polling"
```

Comment:
```suggestion
  "iot_class": "local_polling",
  "loggers": ["aioairzone"]
```

Reply:
Changed in 782937a20b9b1a1b418f28cb489c7793adb24c90